azkaban-common/src/main/java/azkaban/DispatchMethod.java

```diff
@@ -22,4 +22,12 @@ public static DispatchMethod getDispatchMethod(String value) {
   public static boolean isPollMethodEnabled(String dispatchMethod) {
     return DispatchMethod.getDispatchMethod(dispatchMethod) == DispatchMethod.POLL;
   }
+
+  public static boolean isPushMethodEnabled(String dispatchMethod) {
+    return DispatchMethod.getDispatchMethod(dispatchMethod) == DispatchMethod.PUSH;
+  }
+
+  public static boolean isPushContainerizedMethodEnabled(String dispatchMethod) {
+    return DispatchMethod.getDispatchMethod(dispatchMethod) == DispatchMethod.PUSH_CONTAINERIZED;
+  }
```

Comment:
Is it possible to rename the methods to `isXXXMethod` to keep the usage by the caller generic? The caller could use the result of these methods to verify whether a feature is enabled or to validate an input value for example.

Reply:
+1 on that.

Reply:
There is already generic method called getDispatchMethod and where the validation can be done. I am adding more utility methods so that it can directly be used. It will be useful for containerization specifically.